### ISSUE TITLE
Can't "upgrade" local workouts to be saved remotely

### ISSUE THREAD

**Issue by morteako** [sivertschou/dundring#157]

**Comment by morteako** [comment]
Can be discussed if it is a bug.
And even if it is, that it is worth it

**Comment by sivertschou** [comment]
Yeah, we should probably make it possible to both save workouts remotely and locally no matter what you originally did – It might require some more thinking about the workout ids, and the UX when a user creates a workout remotely, makes changes locally and then try to save it remotely – it should at least display a warning of it overwriting the existing workout.
